src/index.ts

```diff
@@ -84,6 +89,15 @@ const finalResult: FinalResult = {
 
 		await tasks.run();
 
+		// Loop over each successful test
+		for (const successfulTest of finalResult.successfulTests) {
+			// Print section header
+			console.log(`\n${chalk.inverse(chalk.white(chalk.bold(` Output from: ${successfulTest.project} `)))}\n`);
```

Comment:
I would expect this to be green, since the failing tests are red. I think this more-closely resembles how Jest's output appears, too.

```suggestion
			console.log(`\n${chalk.inverse(chalk.green(chalk.bold(` Output from: ${successfulTest.project} `)))}\n`);
```